src/storage/storage_handler.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABC, abstractmethod
+
+from src.utils.config import ConfigArguments
+
+class DataStorage(ABC):
```

Comment:
General abstraction should 
DataStorage
- object dir/file/object - node
- path of file/directories - namespace
- file/objects - leaf-node